tests/commands/harddrive.py

```diff
@@ -20,30 +20,35 @@
 
 import unittest
 from tests.baseclass import CommandTest
-from pykickstart.commands.harddrive import FC3_HardDrive
+from pykickstart.commands.harddrive import FC3_HardDrive, F33_HardDrive
+
+
+def attribute_test(data1, data2, test_class):
+    # additional test coverage
+    test_class.assertEqual(data1.__str__(), '')
+
+    # test that new objects are always equal
+    test_class.assertEqual(data1, data2)
+    test_class.assertNotEqual(data1, None)
+
+    # test for objects difference
+    for atr in ['biospart', 'partition', 'dir']:
```

Comment:
The attributes also need to be passed in, that's why the travis test is failing.

Reply:
Fixed.